lib/rules/no-raw-text.ts

```diff
@@ -111,7 +111,7 @@ function checkSvelteLiteralOrText(
   const loc = literal.loc!
   context.report({
     loc,
-    message: `raw text '${literal.value}' is used`
+    message: `raw text '${literal.value.trimStart().trimEnd()}' is used`
```

Comment:
I thought this would help with cleaning up the whitespace so the error message was relevant.